src/components/FileViewer.jsx

```diff
@@ -0,0 +1,38 @@
+/* global cozy */
+
+import utilStyles from '../styles/utils'
+import React from 'react'
+import Loading from './Loading'
+
+class FileViewer extends React.Component {
```

Comment:
The fact this file and object are called `FileViewer` and the fact it requests an `intent` as a props seem inconsistent.
As cozy-drive is a file explorer, I guess we will have a file viewer feature.
In the meantime the file and object's name do not reflect this very feature: service OPEN intent_id